docs/custom_template/partials/footer.tmpl.partial

```diff
@@ -0,0 +1,14 @@
+{{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
+
+<footer>
+  <div class="grad-bottom"></div>
+  <div class="footer">
+    <div class="container">
+      <span class="pull-right">
+        <a href="#top">Back to top</a>
+      </span>
+      {{{_appFooter}}}
+      {{^_appFooter}}<span>Generated by <strong>DocFX</strong></span> | Copyright (c) 2018 The NUnit Project - Licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank">CC BY-NC-SA 4.0</a>{{/_appFooter}}
+    </div>
```

Comment:
Need to ensure that no existing copyright statements are changed without getting permission. Maybe we can come up with a way to automate that. OTOH, each project should be consistent, so may not need to check every page.

Reply:
I verified that all pages actually in the existing Wiki use the same copyright. There are some external links, but since they are external, the copyright isn't an issue.

I have a suspicion about one project, the NUnit VS Test Generator, whose code is copyright to Terje and has never been transferred to the NUnit Project. Of course, the docs are a separate matter and if it's currently wrong, that's not a problem with this issue anyway.

As you set this up, bear in mind that a given project under the NUnit Project could have different licensing and therefore would need a different footer. Currently, there are a few projects, not yet covered by the docs wiki, with a different copyright holder. There's even one (a contribution project) with multiple copyright holders. From what I can tell, it looks like docfx can deal with having separate footers in different branches of the articles tree, so hopefully it won't be a problem if we decide to add any of those.